src/extensions/epigenomics/Encode.ttl

```diff
@@ -18,71 +18,486 @@
-  rdfs:label "no restriction" ;
-  skos:prefLabel "no restriction" ;
+DSPCore:Library
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:cardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:hasLab ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:hasStrandSpecificity ;
+    ] ;
 .
-DSPCore:derivedFrom
-  rdfs:domain DSPCore:Library ;
+DSPCore:SequenceFile
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:usedPlatform ;
+    ] ;
 .
-DSPCore:hasEthnicity
-  rdfs:comment "Use string for now; will be an ontology in future" ;
+DSPCore:hasAssayType
+  rdfs:range obo:OBI_0000070 ;
 .
 <http://datamodel.terra.bio/Encode>
   a owl:Ontology ;
   owl:imports <http://datamodel.terra.bio/DSPCore> ;
   owl:imports <http://datamodel.terra.bio/DSPdcat_ap> ;
-  owl:imports <http://datamodel.terra.bio/OBI_ontofox.owl> ;
   owl:versionInfo "Created with TopBraid Composer" ;
 .
+Encode:AuditFinding
+  a owl:Class ;
+  rdfs:label "AuditFinding" ;
+  rdfs:subClassOf owl:Thing ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:allValuesFrom xsd:integer ;
+      owl:onProperty prov:value ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:cardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:hasLevelName ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:wasFoundIn ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty skos:closeMatch ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty prov:wasGeneratedBy ;
+    ] ;
+.
+Encode:ChIP-seq_assay
+  a obo:OBI_0000716 ;
+  rdfs:label "ChIP-seq assay" ;
+.
+Encode:ChIPSeq
+  a owl:Class ;
+  rdfs:label "ChIPSeq" ;
+  rdfs:subClassOf DSPCore:AssayActivity ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:cardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty DSPCore:usesSample ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:cardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty Encode:hasStrandSpecificity ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty DSPCore:hasAntibody ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty prov:generated ;
+    ] ;
+.
+Encode:ChIPSeqReadMapping
+  a owl:Class ;
+  DSPCore:hasCrossReference "http://encodeproject.org/pipelines/ENCPL220NBH" ;
+  rdfs:label "ChIP-seq read mapping" ;
+  rdfs:subClassOf Encode:Pipeline ;
```

Comment:
I didn't realize there were few enough pipelines that we could enumerate them like this

Reply:
Yes, only about 16, I think.